Gem/Code/Source/MultiplayerSampleTypes.h

```diff
@@ -44,6 +44,48 @@ namespace MultiplayerSample
     };
 
     using RoundTimeSec = AzNetworking::QuantizedValues<1, 2, 0, 3600>; // 1 hour max round duration
+
+    struct PlayerState
+    {
+        AZStd::string m_playerName;
+        uint32_t m_score;          // coins collected
```

Comment:
I would set a default value on these if you have a c-tor that forces assignment for these properties.